--- c-blosc-1.21.2/CMakeLists.txt.orig	2022-12-07 11:34:44.000000000 +0100
+++ c-blosc-1.21.2/CMakeLists.txt	2023-04-27 18:49:32.936930790 +0200
@@ -128,6 +128,8 @@ option(PREFER_EXTERNAL_ZLIB
     "Find and use external Zlib library instead of included sources." OFF)
 option(PREFER_EXTERNAL_ZSTD
     "Find and use external Zstd library instead of included sources." OFF)
+option(REQUIRE_SSE2
+	"Use SSE2 instructions without runtime detection." OFF)
 
 set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake")
 include(GNUInstallDirs)
@@ -276,7 +278,7 @@ endif()
 
 # Set the "-msse2" build flag if supported.
 if(CMAKE_C_COMPILER_ID STREQUAL GNU OR CMAKE_C_COMPILER_ID STREQUAL Clang OR CMAKE_C_COMPILER_ID STREQUAL Intel)
-    if(COMPILER_SUPPORT_SSE2)
+    if(COMPILER_SUPPORT_SSE2 AND REQUIRE_SSE2)
         add_compile_options(-msse2)
     endif()
 endif()
